auth.php

```diff
@@ -74,12 +74,12 @@ protected function _jig($id,$pw,$realm) {
 	protected function _mongo($id,$pw,$realm) {
 		return (bool)
 			$this->mapper->load(
-				array(
+				[
 					$this->args['id']=>$id,
 					$this->args['pw']=>$pw
-				)+
+				]+
 				(isset($this->args['realm'])?
-					array($this->args['realm']=>$realm):array())
+					[$this->args['realm']=>$realm):[]]
```

Comment:
does not look correct. I think it should be
`[$this->args['realm']=>$realm]:[])`